src/AppInstallerCLICore/Commands/ResumeCommand.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "AppInstallerRuntime.h"
+#include "Resources.h"
+#include "ResumeCommand.h"
+#include "RootCommand.h"
+#include "Microsoft/CheckpointIndex.h"
+#include "Workflows/ResumeFlow.h"
+
+namespace AppInstaller::CLI
+{
+    using namespace std::string_view_literals;
+    using namespace Execution;
+
+    std::vector<Argument> ResumeCommand::GetArguments() const
+    {
+        return {
+            Argument::ForType(Execution::Args::Type::ResumeId),
+        };
+    }
+
+    Resource::LocString ResumeCommand::ShortDescription() const
+    {
+        return { Resource::String::ResumeCommandShortDescription };
+    }
+
+    Resource::LocString ResumeCommand::LongDescription() const
+    {
+        return { Resource::String::ResumeCommandLongDescription };
+    }
+
+    Utility::LocIndView ResumeCommand::HelpLink() const
+    {
+        return "https://aka.ms/winget-command-resume"_liv;
+    }
+
+    void ResumeCommand::ExecuteInternal(Execution::Context& context) const
+    {
+        std::string resumeGuidString { context.Args.GetArg(Execution::Args::Type::ResumeId) };
+        if (!Utility::IsValidGuidString(resumeGuidString))
+        {
+            context.Reporter.Error() << Resource::String::InvalidResumeIdError(Utility::LocIndView{ resumeGuidString }) << std::endl;
+            AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_INVALID_RESUME_ID);
+        }
+
+        GUID checkpointId = Utility::ConvertToGuid(resumeGuidString);
+
+        if (!std::filesystem::exists(AppInstaller::Repository::Microsoft::CheckpointIndex::GetCheckpointIndexPath(checkpointId)))
+        {
+            context.Reporter.Error() << Resource::String::ResumeIdNotFoundError(Utility::LocIndView{ resumeGuidString }) << std::endl;
+            AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_RESUME_ID_NOT_FOUND);
+        }
+
+        Execution::Context resumeContext{ std::cout, std::cin };
```

Comment:
Might have to make a new clone-type method on `Context` to properly clone the in/out streams from the incoming context.  But definitely need to inherit them rather than using cout/cin directly.